### ISSUE TITLE
Test new thumbnail height setting

### ISSUE THREAD

**Issue by kimisgold** [omeka/theme-thanksroy#51]
There is now a theme setting for using thumbnails' original height. It sits in the theme's configuration page here:

![image](https://user-images.githubusercontent.com/606764/125688744-ca8cbd4a-419f-4420-9fad-7776d4afca7b.png)

When the checkbox is ticked, it should display images on the browse view according to the values in Appearance > Settings > Thumbnail/Square thumbnail size: 

![image](https://user-images.githubusercontent.com/606764/125689077-dc973a4d-a1f7-43c9-981c-bb9d9beac729.png)

When the checkbox is not ticked, the thumbnails should max out at 100px height.

This should be tested in mobile and desktop browsers. All changes are on the `master` branch.

**Comment by katknow** [comment]
@kimisgold Pulled from master branch, but I can't seem to get the size of the thumbnails to change when I'm checking and unchecking the box. Cleared cache/reset browser too. I have a feeling it's operator error again, but I've re-pulled and no luck.

**Comment by kimisgold** [comment]
@katknow That's odd. I logged into your test site and was able to see a difference when toggling that setting. What browser are you using?

**Comment by katknow** [comment]
@kimisgold I was using Firefox and Safari, but I've uninstalled and reinstalled and the browsers and it seems to be working. I had changed it to 400 px because I was starting think I just wasn't noticing the difference, and now I'm realizing it is really obvious when it changes even between 200 px and 100 px. I recently got a new laptop and did a Mac to Mac transfer from an older machine and things have been buggy so I'm just installing and reinstalling apps and things now and thinking that might fix a lot of the problems I've been having.
